src/main/java/com/library/repository/AuthorRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.library.repository;
+
+import com.library.model.entity.Author;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
```

Comment:
Is it import instruction necessary?

Reply:
For me? Because I didn't split packages?

Reply:
Will discuss on call